templates/user_profile_page.hbs

```diff
@@ -220,119 +241,164 @@
+                      {{#if icon_url}}
+                        <img class="badge" src="{{icon_url}}" alt="">
+                      {{/if}}
+                    </div>
+                    <div>
+                      <div id="title-{{id}}" class="profile-badges-item-title">
                         {{name}}
-                      </a>
-                    {{/each}}
-                  </span>
-                </span>
-              {{/if}}
-            </header>
-
-            <ul class="profile-contribution-list profile-{{current_filter.identifier}}">
-              {{#each contributions}}
+                      </div>
+                      <p id="desc-{{id}}" class="profile-badges-item-description">
+                        {{description}}
+                      </p>
+                    </div>
+                    <div class="profile-badges-item-metadata">
+                      <div class="profile-badges-item-metadata-title">
+                        {{t 'badges_awarded'}}
+                      </div>
+                      <p class="profile-badges-item-metadata-description">
+                        {{date assigned_at format='medium'}}
+                      </p>
+                    </div>
+                  </section>
+                {{/each}}
+              </div>
+            {{else}}
+              <span class="no-activity">{{t 'no_badges'}}</span>
+            {{/if}}
+          </section>
+        </div>
 
-                <li class="profile-contribution">
-                  <span class="profile-contribution-icon" aria-hidden="true">
-                    {{#is object_type "article"}}
-                      <svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" focusable="false" viewBox="0 0 12 12">
-                        <path fill="none" stroke="currentColor" stroke-linecap="round" d="M3.5 5.5h5m-5 2h5m-5 2h5m2-6.29V11c0 .28-.22.5-.5.5H2c-.28 0-.5-.22-.5-.5V1c0-.28.22-.5.5-.5h5.79c.13 0 .26.05.35.15l2.21 2.21c.1.09.15.21.15.35zM7.5.5V3c0 .28.22.5.5.5h2.5"/>
-                      </svg>
-                    {{/is}}
-                    {{#is object_type "post"}}
-                      <svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" focusable="false" viewBox="0 0 12 12">
-                        <path fill="none" stroke="currentColor" stroke-linecap="round" d="M1.5 6.5H1C.7 6.5.5 6.3.5 6V1C.5.7.7.5 1 .5h7c.3 0 .5.2.5.5v2.5m-6-1h4m-1 5h4m-4 2h4M4 5.5h7c.3 0 .5.2.5.5v5c0 .3-.2.5-.5.5H4c-.3 0-.5-.2-.5-.5V6c0-.3.2-.5.5-.5z"/>
-                      </svg>
-                    {{/is}}
-                    {{#is object_type "comment"}}
-                      <svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" focusable="false" viewBox="0 0 12 12">
-                        <path fill="none" stroke="currentColor" d="M1 .5h10c.3 0 .5.2.5.5v7c0 .3-.2.5-.5.5H6l-2.6 2.6c-.3.3-.9.1-.9-.4V8.5H1C.7 8.5.5 8.3.5 8V1C.5.7.7.5 1 .5z"/>
+      {{/is}}
+      {{#isnt current_filter.identifier 'badges'}}
```

Comment:
Following up Augusto comment, because this part is common rendering articles, posts and comments, we can add this here instead:

```hbs
{{#is current_filter.identifier 'activities'}}
   ...
{{/is}}

{{#is current_filter.identifier 'badges'}}
   ...
{{/is}}

{{#isnt current_filter.identifier 'activities'}}
   {{#isnt current_filter.identifier 'badges'}}
      ...
   {{/isnt}}
{{/isnt}}
```